From d56d17175c8309ad80ca76561994ede376b9776a Mon Sep 17 00:00:00 2001
From: Li Wang <li.wang@windriver.com>
Date: Mon, 8 Jul 2013 11:13:10 +0800
Subject: [PATCH 3/7] x86: compile warning: add header file for EXPORT_SYMBOL
 and EXPORT_SYMBOL_GPL

warning: data definition has no type or storage class [enabled by default]
warning: type defaults to 'int' in declaration of 'EXPORT_SYMBOL_GPL' [-Wimplicit-int]
warning: parameter names (without types) in function declaration [enabled by default]

Signed-off-by: Li Wang <li.wang@windriver.com>
---
 arch/x86/kernel/vsyscall_64.c |    1 +
 fs/btrfs/inode.c              |    1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/x86/kernel/vsyscall_64.c b/arch/x86/kernel/vsyscall_64.c
index 7515cf0..bec6828 100644
--- a/arch/x86/kernel/vsyscall_64.c
+++ b/arch/x86/kernel/vsyscall_64.c
@@ -33,6 +33,7 @@
 #include <linux/notifier.h>
 #include <linux/syscalls.h>
 #include <linux/ratelimit.h>
+#include <linux/module.h>
 
 #include <asm/vsyscall.h>
 #include <asm/pgtable.h>
diff --git a/fs/btrfs/inode.c b/fs/btrfs/inode.c
index 9e51325..583c1a9 100644
--- a/fs/btrfs/inode.c
+++ b/fs/btrfs/inode.c
@@ -39,6 +39,7 @@
 #include <linux/slab.h>
 #include <linux/ratelimit.h>
 #include <linux/mount.h>
+#include <linux/module.h>
 #include "compat.h"
 #include "ctree.h"
 #include "disk-io.h"
-- 
1.7.10.4

